src/ExportMap.js

```diff
@@ -525,31 +526,34 @@ ExportMap.parse = function (path, content, context) {
     });
     try {
       if (tsConfigInfo.tsConfigPath !== undefined) {
-        const jsonText = fs.readFileSync(tsConfigInfo.tsConfigPath).toString();
-        if (!parseConfigFileTextToJson) {
-          // this is because projects not using TypeScript won't have typescript installed
-          ({ parseConfigFileTextToJson } = require('typescript'));
-        }
-        return parseConfigFileTextToJson(tsConfigInfo.tsConfigPath, jsonText).config;
+        // Projects not using TypeScript won't have `typescript` installed.
+        if (!ts) (ts = require('typescript'));
+  
+        const configFile = ts.readConfigFile(tsConfigInfo.tsConfigPath, ts.sys.readFile);
+        return ts.parseJsonConfigFileContent(
+          configFile.config,
+          ts.sys,
+          dirname(tsConfigInfo.tsConfigPath),
+        );
       }
     } catch (e) {
       // Catch any errors
     }
-
+  
```

Comment:
```suggestion

```

Reply:
Sorry, was saving without formatting - Prettier was aggressively attacking the file. Resolved!